tests/integration/components/amount-input/component-test.js

```diff
@@ -1,6 +1,6 @@
 import { module, test } from 'qunit'
 import { setupRenderingTest } from 'ember-qunit'
-import { blur, fillIn, render, typeIn } from '@ember/test-helpers'
+import { blur, fillIn, render, typeIn, pauseTest } from '@ember/test-helpers'
```

Comment:
Can remove unneeded import, btw you can access pauseTest directly on the test context without an import `await this.pauseTest()`